tests/yam/agama/auto/playwright_test.pm

```diff
@@ -0,0 +1,22 @@
+## Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Run playwright tests directly from the Live ISO.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base 'y2_installbase';
+use strict;
+use warnings;
+
+use testapi qw(assert_screen assert_script_run get_var select_console);
```

Comment:
it is much more readable for me like this:
```
use testapi qw(
  assert_screen
  assert_script_run
  get_var
  select_console
);
```
wdyt?

Reply:
updated